compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -915,12 +915,25 @@ pub struct CoercePredicate<'tcx> {
 }
 pub type PolyCoercePredicate<'tcx> = ty::Binder<'tcx, CoercePredicate<'tcx>>;
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct Term<'tcx> {
     ptr: NonZeroUsize,
     marker: PhantomData<(Ty<'tcx>, Const<'tcx>)>,
 }
 
+impl Debug for Term<'_> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let data = if let Some(ty) = self.ty() {
```

Comment:
this is a nit but shouldn't this just match on `self.unpack()`

Reply:
xd I didn't know that method existed

Reply:
Shouldn't this just print `unpack()`?

Reply:
https://doc.rust-lang.org/beta/nightly-rustc/src/rustc_middle/ty/subst.rs.html#101-109 for reference